### ISSUE TITLE
<tab> autocompletion not working properly for bash

### ISSUE THREAD

**Issue by eriksholmlund** [austinjones/tab-rs#157]
Pressing <tab> shows the correct tab list as suggestions, but autocompleting with <tab> either works unexpectedly or doesn't work at all.

How I installed tab:
```bash
cargo install tab

tab --install bash
```
in a `tab.yml` in one of my projects I have the following:
```yaml
repo: testrepo
doc: "my project"

tabs:
        - tab: inputs
          dir: input/
          doc: "input folder"
```
when I `cd` into the project, I can accurately press `tab tes<tab>` to autocomplete to `tab testrepo/` but when I try `tab testrepo/inp<tab>` it corrects me back to `tab testrepo/`. If I add another tab, e.g. `tab anotherrepo`, now my <tab> autocompletion doesn't work at all, and only shows me the available options if I press <tab> twice.

A workaround that is completely fine for me is to change the bash completion script to:
```bash
function _tab() {
        local cur opts
        COMPREPLY=()
        cur="${COMP_WORDS[COMP_CWORD]}"
        opts=$(tab -l | tail -n +2 | cut -d "(" -f 1 | sed "s/ //g")
  
        COMPREPLY=( $(compgen -W "$opts" -- ${cur}) )
        return 0
  }
  
  complete -F _tab tab
```
where I am basically just reformatting the `tab -l` output.

Otherwise, this is a great tool. Thank you!

**Comment by austinjones** [comment]
Hi @eriksholmlund,
Thanks for the great bug report!  I took a closer look and my bash script definitely needs some work.

**Comment by austinjones** [comment]
I pushed a release with a fix (#158).  It should work now with `cargo install tab`, and `tab --install bash`.

**Comment by eriksholmlund** [comment]
Hi @austinjones,
It works perfectly now with default settings. Thanks for the quick fix!
